.github/workflows/test_osdk.yml

```diff
@@ -10,9 +10,13 @@ on:
   push:
     branches:
       - main
+  # Schedule to run on every day at 15:30 UTC (23:30 Beijing Time)
+  schedule:
+    - cron: '30 15 * * *'
```

Comment:
According to [BandwidthPlace](https://www.bandwidthplace.com/article/how-does-time-of-day-affect-internet-speed), early in morning has the least internet traffic in a day.

> Early Morning (4 AM – 7 AM): You’ll likely experience the fastest speeds due to minimal traffic, as most people are still asleep.
Morning-Afternoon (7 AM – 4 PM): Expect slight fluctuations in speed, particularly during lunch hours. This is when many people at work use their devices to catch up on news, social media, or personal tasks.
Evening (4 PM – 10 PM): Often referred to as “internet peak hours,” evenings are when the highest number of users are online. This can lead to notably slower speeds, especially for streaming and gaming.
Late Night (10 PM – 4 AM): As people start heading to bed, internet traffic reduces, and speeds begin to improve.

So we should choose a time between 4 AM – 7 AM to trigger scheduled CI.

Please also update benchmark CI and TDX Asterinas CI if they are not scheduled between 4 AM - 7 AM, Beijing time already. And they should not be run all at once. For example, choose 4AM, 5AM, and 6AM, respectively.